pkg/operator/staticpod/controller/prune/prune_controller.go

```diff
@@ -186,11 +185,14 @@ func (c *PruneController) pruneAPIResources(ctx context.Context, toKeep sets.Int
 	return nil
 }
 
+//go:embed manifests/pruner-pod.yaml
+var template []byte
```

Comment:
`template` => `podTemplate`, what do you think?

Reply:
+1, done.